src/screens/DiscoveryScreenV2/index.js

```diff
@@ -336,9 +343,3 @@ DiscoveryScreenV2.propTypes = {
 };
 
 export default withInteractionsManagedNoStatusBar(DiscoveryScreenV2);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `DiscoveryContainer` component has been removed from the end of the file. As mentioned earlier, this is a good practice for readability and maintainability. Just ensure that there are no references to this component after its previous position in the codebase to avoid potential ReferenceErrors.
```diff
- const DiscoveryContainer = ({children}) => {
-   if (Platform.OS === 'ios') return <>{children}</>;
- 
-   return <SafeAreaView style={{flex: 1}}>{children}</SafeAreaView>;
- };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->